lib/ff_syscall_wrapper.c

```diff
@@ -610,6 +610,11 @@ ff_sendmsg(int s, const struct msghdr *msg, int flags)
 {
     int rc;
 
+    if (msg->msg_name != NULL) {
+        linux2freebsd_sockaddr(msg->msg_name,
```

Comment:
`msg`'s value has been modified, it's against with the const qualifier.
Better like this:
```
void *oldsa = msg->msg_name;
struct sockaddr sa;
linux2freebsd_sockaddr(msg->msg_name, sizeof(struct linux_sockaddr), &sa);
msg->msg_name = &sa;
sendit..
msg->msg_name = oldsa;
```